### DISCUSSION TITLE
Create dialog-like window

### DISCUSSION THREAD

**Discussion by SamuelC0507** [tauri-apps/tauri#6569] [Q&A] [answered]
As in `.NET` we'd use `Form.ShowDialog` to open a new child window if we wish to force user to close it before having access to the parent window again. Is there a way to create a new window in Tauri that emulates the similar behavior? I know Tauri can open dialog window but that's limited to showing messages. Simply put, is there a way to have the blocking effect of a dialog on a normal window?

Thanks!

**Comment by FabianLars** [comment, marked as answer]
Tauri does have some owner and parent APIs (in rust) but i don't think they implement a blocking behavior on all Platforms 🤔 
https://docs.rs/tauri/latest/x86_64-pc-windows-msvc/tauri/window/struct.WindowBuilder.html#method.parent_window / https://docs.rs/tauri/latest/x86_64-pc-windows-msvc/tauri/window/struct.WindowBuilder.html#method.owner_window (macos only has a parent_window method, and linux has neither)

The closest feature request in this repo seems to be this one: https://github.com/tauri-apps/tauri/issues/1643

**Reply by SamuelC0507** [reply]
Understood. Thanks for the information!
